packages/vscode-integration/src/quickstart-components/glsp-server-adapter.ts

```diff
@@ -0,0 +1,114 @@
+/********************************************************************************
+ * Copyright (c) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import * as net from 'net';
+import * as vscode from 'vscode';
+import { createMessageConnection, SocketMessageReader, SocketMessageWriter } from 'vscode-jsonrpc';
+import { ApplicationIdProvider, BaseJsonrpcGLSPClient } from '@eclipse-glsp/protocol';
+import { isActionMessage } from '../actions';
+
+import { GlspVscodeServer } from '../types';
+
+interface Options {
```

Comment:
IMO too generic. The naming should reflect the option type (e.g. `GlspServerAdapterOptions`)

Reply:
Renamed in 26e430709fab8d764971bf477ba694e2595aa6a2